utils/helpers.py

```
@@ -0,0 +1,5 @@
+def str_to_bool(bool_str):
+    if bool_str == "True":
+        return True
+    else:
+        return False
```

Ruff

Return the condition `bool_str == "True"` directly

Replace with `return bool_str == "True"`

(SIM103)